### ISSUE TITLE
Ability to bind events on mouse clicks

### ISSUE THREAD

**Issue by josser** [jigish/slate#361]
Hi, i'm not sure it is polite to ask about such things, and I know this is not commercial project.
I have a mouse with 6 buttons and It would be cool to bind some actions on some unused keys. 

For example there is a button named "Aim", it can be used to make app full screen. 
Also, it would be great to bind some actions on keyboard and mouse combos like Shift + 'mouse4 (back)' to move window to the left screen. 

Thank you and sorry for my (maybe) not good english.

**Comment by larrybolt** [comment]
What you could do is use BetterTouchTool to and bind actions to mouse button actions, BetterTouchTool itself has also actions to make windows fullscreen or to move them around.
http://lry.be/l0nD/1EhZ08Zk

I also think the company of that mouse will have made custom drivers available where you can bind each individual button to actions or modifiers/keypresses.

**Comment by josser** [comment]
Yep, this is MadCatz R.A.T 9 mouse but native mac software suck a lot. It have possibility to bind some keystrokes but this functions just don't work. 
Anyway, I will try BetterTouchTool. Thanks!

**Comment by jedahan** [comment]
I would love this to work for events like dragging a window to either side of the screen, like in http://www.irradiatedsoftware.com/cinch/

**Comment by mathias** [comment]
:+1: for extra mouse button support. I stopped using BetterTouchTool in favor of Slate for window management because BetterTouchTool was getting flakey and seemed to be breaking many common app key combos for me.
